src/version/properties.js

```diff
@@ -0,0 +1,51 @@
+const BaseVersioning = require('./base')
+const bumpVersion = require('../helpers/bumpVersion')
+
+module.exports = class Properties extends BaseVersioning {
+
+  fileContent = null
+
+  #getProperty(content, key) {
+    const regex = new RegExp(`^\\s*${key}\\s*[:=]\\s*(.*)$`, "m")
+    const match = content.match(regex)
+    return match ? match[1].trim() : null
+  }
+
+  #setProperty(content, key, value) {
+    // https://regexr.com/8gtpa
+    const regex = new RegExp(`^(\\s*${key}\\s*)([:=])(.*)$`, "m")
+
+    if (regex.test(content)) {
+      return content.replace(regex, (_, _key, sep) => `${key.trim()}${sep.trim()}${value}`)
+    }
+    else {
+      if (content.length > 0 && !content.endsWith("\n")) {
+        return content + `\n${key}=${value}`
+      }
+      return content + `${key}=${value}`
+    }
+  }
```

Comment:
Could you align this code to the rest of the code style?

Reply:
Which code style do you use? I normally don't code in js and I have a hard time seeing any difference.

Reply:
If you remove the `#` from the function names i'm ok with it.